JustSaying.TestingFramework/MessageStubs.cs

```diff
@@ -18,6 +18,14 @@ public class AnotherGenericMessage : Message
         public string Content { get; set; }
     }
 
+    public class DelayedMessage : Message
+    {
+        public DelayedMessage(int delaySeconds)
+        {
+            DelaySeconds = delaySeconds;
```

Comment:
http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html

Valid values: 0 to 900. Maximum: 15 minutes.